astropy/units/quantity_helper/function_helpers.py

```diff
@@ -483,8 +481,7 @@ def where(condition, *args):
 
 
 # Quantile was only introduced in numpy 1.15.
```

Comment:
This comment reminds me that there are similar ones in `helpers.py` where for some reason `NUMPY_LT_*` is not used. If you are up to it, do change that too, but feel free to tell me to do it (and I'll make a PR to yours).